plugins/scaffolder-backend/src/scaffolder/tasks/StorageTaskBroker.ts

```diff
@@ -262,6 +264,20 @@ export class StorageTaskBroker implements TaskBroker {
     );
   }
 
+  /**
+   * {@inheritdoc TaskBroker.closeStaleTasks}
+   */
+  async closeStaleTasks(): Promise<void> {
+    const timeoutS =
+      this.config.getOptionalNumber('scaffolder.taskTimeout.seconds') || 300;
```

Comment:
Tbh it might make sense to just use some sensible default for this and skipping the possibility to configure it? I'm thinking 1h or so is well over any reasonable task duration but it'll still clean things up pretty quick.

Reply:
Makes sense, I was expecting some feedback on that. 
I'll remove that configuration 😉